splinterd/man/splinterd.1.md

```diff
@@ -112,6 +112,10 @@ OPTIONS
   Any options on the command line will override the settings in the
   configuration file.
 
+`--config-dir CONFIG_DIR`
+: Specifies the directory containing configuration files.
+  (Default: `/etc/splinter`.)
```

Comment:
Make consistent with other options:
- `CONFIG-DIR` (with a hyphen)
- Mention the `SPLINTER_CONFIG_DIR` environment variable here, following the model of `--tls-ca-file` (precede with a blank line and indent so it lines up with the existing description).

Reply:
Updated.